reporter/src/main/kotlin/reporters/evaluatedmodel/StatisticsCalculator.kt

```diff
@@ -78,6 +79,10 @@ internal class StatisticsCalculator {
         )
     }
 
+    private fun getOpenVulnerabilities(ortResult: OrtResult, resolutionProvider: ResolutionProvider): Int =
+        ortResult.getVulnerabilities(omitExcluded = true).values.flatten()
```

Comment:
Just a question for my understanding: In this implementation resolved vulnerabilities are filtered out using the `resolutionProvider`. It would also be possible to call `getVulnerabilities()` with `omitResolved` set to *true*, right?. What is the difference? And isn't there some redundancy with the different ways to apply resolutions?

Reply:
The difference is that for `omitResolved` the function only takes vulnerability resolutions into account that are configured in the repository configuration. The `resolutionProvider` also knows about global vulnerability resolutions. I'm not super happy with that, but I have included the option to be consistent with the `getRuleViolations()` function.

Reply:
Thanks for explaining.